src/pages/UserLogin.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import api from '../api'
+
+import styled from 'styled-components'
+
+const Title = styled.h1.attrs({
+    className: 'h1',
+})``
+
+const Wrapper = styled.div.attrs({
+    className: 'form-group',
+})`
+    margin: 0 800px;
+`
+
+const Label = styled.label`
+    margin: 5px;
+`
+
+const InputText = styled.input.attrs({
+    className: 'form-control',
+})`
+    margin: 5px;
+`
+
+const Button = styled.button.attrs({
+    className: `btn btn-primary`,
+})`
+    margin: 15px 15px 15px 188px;
+`
+
+class UserLogin extends Component {
```

Comment:
Please use functional approach to creating components  with Hooks - [link](https://uk.reactjs.org/docs/hooks-intro.html)